node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -58,39 +58,27 @@
     }
 }
 
-/* MediaCore preview */
-
-#filepreview-container #filepreview-mediacore,
+#filepreview-container #filepreview-video-container,
 #filepreview-container #filepreview-markdown-container {
     padding: 20px 0;
 }
 
-#filepreview-container #filepreview-mediacore iframe {
-    width: 100%;
-}
-
-.mobile #filepreview-container #filepreview-mediacore iframe {
-    width: 340px; /* iOS will need an explicit width for the iframe */
-}
-
-#filepreview-container #filepreview-mediacore.filepreview-mediacore-audio iframe {
-    height: 24px;
-}
+/* Video preview */
```

Comment:
Do you know if there's a way to make `video.js` fill the available player space, similar to how YouTube embedding works?

<img width="1779" alt="screen shot 2016-08-01 at 11 34 55" src="https://cloud.githubusercontent.com/assets/109850/17304764/1263aa5a-57dc-11e6-800b-a203b18b3a81.png">